Comparison Engine/Comparison Engine/Child Forms/BarForm.cs

```diff
@@ -12,44 +13,69 @@ namespace Comparison_Engine.Forms
 {
     public partial class BarForm : Form
     {
-        public BarForm(Bar bar)
+        public Bar bar = null;
+        public DrinkManager drinkManager = null;
```

Comment:
Drink Manager is a singleton, so why would you ever use anything but the instance of it?